Sources/StreamChat/APIClient/Endpoints/EndpointPath.swift

```diff
@@ -0,0 +1,87 @@
+    case showChannel(String, Bool)
+    case truncateChannel(String)
+    case markChannelRead(String)
+    case markAllChannelsRead
+    case channelEvent(String)
+    case stopWatchingChannel(String)
+    case pinnedMessages(String)
+    case uploadAttachment(String, String)
+
+    case sendMessage(ChannelId)
+    case message(MessageId)
+    case editMessage(MessageId)
+    case deleteMessage(MessageId)
+    case replies(MessageId)
+    case reactions(MessageId)
+    case addReaction(MessageId)
+    case deleteReaction(MessageId, MessageReactionType)
+    case messageAction(MessageId)
+
+    case banMember
+    case flagUser(Bool)
+    case flagMessage(Bool)
+    case muteUser(Bool)
+
+    var value: String {
+        switch self {
+        case .connect: return "connect"
+        case .sync: return "sync"
+        case .users: return "users"
+        case .guest: return "guest"
+        case .members: return "members"
+        case .search: return "search"
+        case .devices: return "devices"
+
+        case .channels: return "channels"
+        case let .createChannel(queryString): return "channels/\(queryString)/query"
+        case let .updateChannel(queryString): return "channels/\(queryString)/query"
+        case let .deleteChannel(payloadPath): return "channels/\(payloadPath)"
+        case let .channelUpdate(payloadPath): return "channels/\(payloadPath)"
+        case let .muteChannel(mute): return "moderation/\(mute ? "mute" : "unmute")/channel"
+        case let .showChannel(channelId, show): return "channels/\(channelId)/\(show ? "show" : "hide")"
+        case let .truncateChannel(channelId): return "channels/\(channelId)/truncate"
+        case let .markChannelRead(channelId): return "channels/\(channelId)/read"
+        case .markAllChannelsRead: return "channels/read"
+        case let .channelEvent(channelId): return "channels/\(channelId)/event"
+        case let .stopWatchingChannel(channelId): return "channels/\(channelId)/stop-watching"
+        case let .pinnedMessages(channelId): return "channels/\(channelId)/pinned_messages"
+        case let .uploadAttachment(channelId, type): return "channels/\(channelId)/\(type)"
+
+        case let .sendMessage(channelId): return "channels/\(channelId.apiPath)/message"
+        case let .message(messageId): return "messages/\(messageId)"
+        case let .editMessage(messageId): return "messages/\(messageId)"
+        case let .deleteMessage(messageId): return "messages/\(messageId)"
+        case let .replies(messageId): return "messages/\(messageId)/replies"
+        case let .reactions(messageId): return "messages/\(messageId)/reactions"
+        case let .addReaction(messageId): return "messages/\(messageId)/reaction"
+        case let .deleteReaction(messageId, reaction): return "messages/\(messageId)/reaction/\(reaction.rawValue)"
+        case let .messageAction(messageId): return "messages/\(messageId)/action"
+
+        case .banMember: return "moderation/ban"
```

Comment:
Don't we have unban member?

Reply:
It uses the same path, different parameters

Reply:
https://github.com/GetStream/stream-chat-swift/pull/1810/files#diff-887fd13025bf4b9e693ada9eec5f554291c79eca1b8c83a54ad8ffb717e749e2R33